pybamm/models/submodels/electrolyte.py

```diff
@@ -3,180 +3,61 @@
-    #     elif domain == "xcp":
-    #         gamma_dl = self.subparam.gamma_dl_p
-    #
-    #     dedt = 1 / gamma_dl * (self.operators[domain].div(i) - j)
-    #
-    #     return dedt
-    #
-    # def macinnes(self, domain, c, e, current_bcs):
-    #     """MacInnes equation for the electrolyte current density.
-    #
-    #     Parameters
-    #     ----------
-    #     domain : string
-    #         The domain in which to calculate the electrolyte current density.
-    #     c : array_like, shape (n,)
-    #         The electrolyte concentration.
-    #     e : array_like, shape (n,)
-    #         The potential difference.
-    #     current_bcs : 2-tuple of array_like, shape (1,)
-    #         Flux at the boundaries (Neumann BC).
-    #
-    #     Returns
-    #     -------
-    #     i : array_like, shape (n+1,)
-    #         The current density.
-    #     """
-    #     operators = self.operators[domain]
-    #     kappa_over_c = 1
-    #     kappa = 1
-    #
-    #     # Calculate inner current
-    #     i_inner = kappa_over_c * operators.grad(c) + kappa * operators.grad(e)
-    #
-    #     # Add boundary conditions
-    #     lbc, rbc = current_bcs
-    #     i = np.concatenate([lbc, i_inner, rbc])
-    #
-    #     return i
-    #
-    # def bcs_current(self, domain, t):
-    #     """Boundary conditions for the current conservation equation.
-    #
-    #     Returns
-    #     -------
-    #     2-tuple of array_like, shape(1,)
-    #         The boundary conditions.
-    #
-    #     """
-    #     if domain == "xcn":
-    #         current_bc_left = np.array([0])
-    #         current_bc_right = np.array([self.subparam.icell(t)])
-    #     elif domain == "xcp":
-    #         current_bc_left = np.array([self.subparam.icell(t)])
-    #         current_bc_right = np.array([0])
-    #     return (current_bc_left, current_bc_right)
+import pybamm
+
+
+class StefanMaxwellDiffusion(pybamm.BaseModel):
+    """A class that generates the expression tree for Stefan-Maxwell Diffusion in the
+    electrolyte.
+
+    Attributes
+    ----------
+
+    rhs: dict
+        A dictionary that maps expressions (variables) to expressions that represent
+        the rhs
+    initial_conditions: dict
+        A dictionary that maps expressions (variables) to expressions that represent
+        the initial conditions
+    boundary_conditions: dict
+        A dictionary that maps expressions (variables) to expressions that represent
+        the boundary conditions
+    variables: dict
+        A dictionary that maps strings to expressions that represent
+        the useful variables
+
+    *Extends:* :class:`BaseModel`
+    """
+
+    def __init__(self, G):
+        super().__init__()
+
+        electrolyte_domain = [
+            "negative electrode",
+            "separator",
+            "positive " "electrode",
+        ]
+
+        c_e = pybamm.Variable("c_e", domain=electrolyte_domain)
+
+        delta = pybamm.Parameter("delta")
```

Comment:
I'm not sure what these parameters are, but if they don't appear as "raw" parameters in a parameter csv file they need to be defined as a combination of raw parameters here. e.g. 
```python3
a = Parameter("something in the csv file")
b = Parameter("something else in the csv file")
delta = a / b
```
If such "combined" parameters are standard battery parameters, they can be defined in and imported from the module `standard_parameters` to avoid having to repeatedly define them in submodels. See issue #37